### ISSUE TITLE
process FCC-hh LHEs with pythia+delphes (+rivet)

### ISSUE THREAD

**Issue by jstupak** [Snowmass21-software/MCProd#10]

**Comment by trtomei** [comment]
Where are these FCC-hh LHEs?

**Comment by jstupak** [comment]
Awesome, thanks!  Naively, I think this should be "easy."  You could use Andrew's job submission and monitoring code to do this (just change madgraph to only run pythia on the existing LHEs).  But that might be overkill for this?

The events can be found here:
http://fcc-physics-events.web.cern.ch/fcc-physics-events/LHEevents.php
Access to these samples requires joining a particular egroup.  I moved a subset of the samples to the OSG machine: /collab/project/snowmass21/data/fcc-hh

**Comment by trtomei** [comment]
I believe that "changing madgraph to only run pythia" seems a bit overkill indeed, but I have pinged Andrew in the Slack. Maybe it is easier to just run the MG5-PY8 interface manually, I don't know.

**Comment by jstupak** [comment]
that might be true.  But we need to find **a way** to run pythia on existing LHEs, becuase it seems like pythia is not run automatically when producing events from a gridpack, even if you set shower=Pythia8 when you generate the gridpack.  I very briefly googled this and it seemed like it was not difficult, but the devil is always in the details: https://answers.launchpad.net/mg5amcnlo/+question/231011

**Comment by PerilousApricot** [comment]
What I really need is some sort of setup script and then a command to run to actually execute pythia

**Comment by PerilousApricot** [comment]
e.g. `/home/meloam/snowmass-ef-mc-submitter/snowmassefmcsubmitter/dag.py` on the snowmass machine, lines 325-358

**Comment by jstupak** [comment]
Yes, this is what we are trying to figure out.  None of us have done this previously, but the launchpad link above is related to this, so I was hoping the answer was there somewhere.  I just haven;t had a chance to fully read it yet

**Comment by jstupak** [comment]
> /home/meloam/snowmass-ef-mc-submitter/snowmassefmcsubmitter/dag.py

permission denied

**Comment by PerilousApricot** [comment]
grr, try now

**Comment by jstupak** [comment]
BTW @PerilousApricot when you have a chance it would be great to put the submission/monitoring code in our github organization.  But there are more pressing to do's

**Comment by jstupak** [comment]
so this is the "old" workflow where we ran pythia within delphes.  It should be trivial to adapt this to skip the matrix element generation; just skip madgraph.  This would allow us to do an apples to apples comparison of our existing samples with the Fcc-hh samples.

Moving forward though, it is not so simple in the "new" workflow where we run pythia within madgraph (and it is therefore not possible to just skip madgraph entirely)

**Comment by trtomei** [comment]
On the other hand, this "new" workflow may not be good after all, since if we look in the other issue (https://github.com/Snowmass21-software/MCProd/issues/1), it seems that even the MadGraph authors do not support running using the Py8 integration when running from a gridpack :(
